### DISCUSSION TITLE
Representative ID

### DISCUSSION THREAD

**Discussion by freshface** [fw4-bvba/whise-api#28] [General] [answered]
Does whise return the representative on a single estate?

**Comment by fyrts** [comment]
Is this concerning the [WebsiteDesigner](https://api.whise.eu/WebsiteDesigner.html) API, or the [EstateBasedService](https://api.whise.eu/EstateBasedService.html) API? I have no experience with the EstateBasedService API, but judging from the documentation all API's should return representative information for estates.

**Comment by freshface** [comment, marked as answer]
Found it, i added ```'ShowRepresentatives' => true``` to to the filter.
